tools/cloudharness_utilities/codefresh.py

```diff
@@ -224,9 +224,9 @@ def create_k8s_rollout_commands():
             for app_name, app in helm_values.apps.items():
                 if app.harness.secrets:
                     for secret in [secret[0] for secret in app.harness.secrets.items() if secret[1]]:
-                        secret_name = secret
+                        secret_name = secret.replace("_", "__")
                         arguments["custom_values"].append(
-                            "apps.%s.harness.secrets.%s=${{%s}}" % (app_name, secret_name, secret_name.upper()))
+                            "apps.%s.harness.secrets.%s=${{%s}}" % (app_name.replace("_", "__"), secret_name, secret_name.upper()))
```

Comment:
@filippomc I think this is not correct, the `.` should be `_` 
see https://github.com/MetaCell/cloud-harness/blob/3483d3d171c7e77ed5a9eec7f4829a86e2902ccd/utilities/cloudharness_utilities/codefresh.py#L126

Reply:
I'm not following

Reply:
Dots can be used in the new codefresh step spec https://codefresh.io/docs/docs/new-helm/using-helm-in-codefresh-pipeline/

Reply:
but there's only one example using the dot, deploying a pipeline to see if that works

Reply:
here I read that you need to replace dots by `_` https://codefresh.io/docs/docs/new-helm/using-helm-in-codefresh-pipeline/

Reply:
Can tell with certainty that the . works but I see the point as the _ is mentioned in the docs as the option. Changed.